jung-visualization/pom.xml

```diff
@@ -14,6 +14,12 @@
     <version.logback>1.2.3</version.logback>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>com.google.truth</groupId>
+      <artifactId>truth</artifactId>
+      <version>0.39</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I think that the version information for Truth here (i.e. the fact that we're using version `0.39`), as well as the version info in all other poms where you introduced Truth, could be externalised to the root `pom.xml`. So I'd expect to see the following property:

```xml
<truth.version>0.39</truth.version>
```

added just right underneath [these lines](https://github.com/jrtom/jung/blob/master/pom.xml#L68-L70) in the root pom, and then replacing all usages of `0.39` in the child poms with `${truth.version}`. So where you would have had the following sort of line previously:

```xml
<version>0.39</version>
```

it would now look like:

```xml
<version>${truth.version}</version>
```

Have I described myself well?

Reply:
I figured that would come in via your CTree test PR and I'd save you a merge conflict

Reply:
Oh, cool, I hadn't considered that, thank you very much! Let's deal with this then after this PR and the CTree one are merged. :)